src/execute_machine_handler.erl

```diff
@@ -1,6 +1,6 @@
 -module(execute_machine_handler).
 -behaviour(cowboy_handler).
--export([init/2, tape_history_accumulator_process/1]).
+-export([init/2, tape_history_accumulator_process/1, execute_machine/2]).
```

Comment:
I'm wondering if we wanna keep the execute_machine/2 function in this file :thinking:. I think yes, what's your opinion ?